java/src/main/java/com/google/appengine/tools/mapreduce/impl/shardedjob/ShardedJobRunner.java

```diff
@@ -90,12 +94,18 @@ public ShardedJobRunner(
     this.datastore = datastore;
     this.appEngineServicesService = appEngineServicesService;
     this.taskQueue = taskQueue;
+    if (System.getProperty("GOOGLE_CLOUD_PROJECT") != null) {
+      // assume non testing environment
+      DELAY_MULTIPLIER = 5;
```

Comment:
not cleanest way of doing this... but didn't want to mess with injection just for this particular case, seemed simple enough for testing